project/plugins.sbt

```diff
@@ -2,14 +2,14 @@ addSbtPlugin("com.cavorite" % "sbt-avro-1-8" % "1.1.3")
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
 addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.1")
 addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.3")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.7")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
+addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.8")
+addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.1")
```

Comment:
IIRC one of the plugin upgrades broken console and made it impossible to enter passphrase when releasing. Don't remember which exactly but probably `sbt-pgp`.

Reply:
@nevillelyh verified that this version works

Reply:
Nevermind it was 1.0.1, so I guess it's fixed.
https://github.com/spotify/scio/commit/cf5386339a7cf67a7e3e74e9706b5a97929c5c95#diff-a84f91f20f040218bccd09fed4761fb3